module-1/web-project/your-code/API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
The project asks for saving the results of API and scraping section in an output folder. You have missed the last part of the exercises!!